draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1204,16 +1203,15 @@ multiple invocations of hash\_to\_base, by factoring out the common computation.
 
 ## Implementation {#hashtobase-impl}
 
-The following procedure implements hash\_to\_base.
+The following procedure implements hash\_to\_base. It requires HKDF-Expand
+from {{!RFC5869}}, which is computed using input hash function H.
```

Comment:
Suggest removing the second sentence ("It requires...") and instead defining HKDF-Expand-H in the Parameters section below.

Reply:
Yep, that's cleaner! Will remove.